content/en/docs/contribute/style/style-guide.md

```diff
@@ -116,17 +116,20 @@ The copy is called a "fork". | The copy is called a "fork."
 
 ## Inline code formatting
 
-### Use code style for inline code, commands, and API objects {#code-style-inline-code}
+### Use code style for inline code, commands {#code-style-inline-code}
 
 For inline code in an HTML document, use the `<code>` tag. In a Markdown
-document, use the backtick (`` ` ``).
+document, use the backtick (`` ` ``). However, API kinds such as StatefulSet
```

Comment:
```suggestion
For inline code in an HTML document, surround it with the `<code>` `</code>` tag. In a Markdown
```
I think that's what @kbhawkey is talking about . Right?

Reply:
Sure,agreed.
For a purely new contributor the explanation of both opening and closing tag  will be helpful but for someone who is some experience in HTML will know this simple logic.

Reply:
The text:
> ```
> document, use the backtick (`` ` ``)
> ```

is not changed in this PR. Because it's part of what I'm not changing, I propose we don't need to discuss that part much further. Does that sound reasonable?

Reply:
sure